routers/user.controller.js

```diff
@@ -52,7 +52,9 @@ class UserController {
 
       const token = await foundUser.generateAndSaveToken();
 
-      const { _id, name, familyId } = foundUser;
+      const { _id, name } = foundUser;
+
+      const familyId = await userModel.findOne({ _id }).populate('familyId');
```

Comment:
you can add populate in first request to DB instead of doing it twice. So you shouldn't delete familyId from foundedUser